feat(hooks): expose DOM components (#3450)
algolia/react-instantsearch: packages/react-instantsearch-hooks-web/README.md

# react-instantsearch-hooks-web

> 🚧 This version is not yet production-ready.

> React InstantSearch Hooks is an open-source, **experimental UI library** for React that lets you quickly build a search interface in your front-end application.

Author: the experimental fits here, until we have the docs, right? or would you prefer it seeing removed and changed to beta before release?

Reviewer: If we're shipping the doc before the release announcement, I don't mind the warning being removed now.

Author: ```suggestion
React InstantSearch Hooks is an open-source, UI library for React that lets you quickly build a search interface in your front-end application.
```

Reviewer 2: I believe we're good removing the experimental version because we've already removed the warning inside the library.

InstantSearch’s goal is to help you implement awesome search experiences as smoothly as possible by providing a [complete search ecosystem](https://algolia.com/doc/guides/getting-started/how-algolia-works/#the-full-ecosystem). InstantSearch tackles an important part of this vast goal by providing front-end primitives that you can assemble into unique search interfaces.

<p align="center">
  <a href="https://codesandbox.io/s/github/algolia/react-instantsearch/tree/master/examples/hooks" title="Edit on CodeSandbox">
    <img alt="Edit on CodeSandbox" src="https://codesandbox.io/static/img/play-codesandbox.svg">
  </a>
</p>

## Installation

React InstantSearch Hooks is available on the npm registry. It relies on [`algoliasearch`](https://github.com/algolia/algoliasearch-client-javascript) to communicate with Algolia APIs.

```sh

> yarn add algoliasearch react-instantsearch-hooks
> # or
> npm install algoliasearch react-instantsearch-hooks

Reviewer: ```suggestion
yarn add algoliasearch react-instantsearch-hooks-web
# or
npm install algoliasearch react-instantsearch-hooks-web
```

# react-instantsearch-hooks-web

> React InstantSearch Web is a React library that lets you create an instant-search result experience using [Algolia][algolia-website]’s search API. It is part of the InstantSearch family:

Reviewer 3: ```suggestion
React InstantSearch Hooks Web is an open-source React library that lets you create an instant search result experience using [Algolia][algolia-website]’s search API. It is part of the InstantSearch family:
```

**React InstantSearch** | [InstantSearch.js][instantsearch.js-github] | [Angular InstantSearch][instantsearch-angular-github] | [Vue InstantSearch][instantsearch-vue-github] | [InstantSearch Android][instantsearch-android-github] | [InstantSearch iOS][instantsearch-ios-github]

## Why

> You should be using React InstantSearch if you want to:

Reviewer 3: ```suggestion
You should be using React InstantSearch Hooks Web if you want to:
```

* Design search experiences with best practices
* Customize your components at will
* Follow React principles

## Installation

React InstantSearch is available on the npm registry. It relies on [`algoliasearch`](https://github.com/algolia/algoliasearch-client-javascript) to communicate with Algolia APIs.

```sh

> yarn add algoliasearch react-instantsearch-web
> # or
> npm install algoliasearch react-instantsearch-web

Reviewer: ```suggestion
yarn add algoliasearch react-instantsearch-hooks-web
# or
npm install algoliasearch react-instantsearch-hooks-web
```

```

## Getting started

> Using React InstantSearch is as simple as adding these components to your app:

Reviewer 3: ```suggestion
Using React InstantSearch Hooks Web is as simple as adding these components to your app:
```

```javascript
import React from 'react';
import ReactDOM from 'react-dom';
import algoliasearch from 'algoliasearch/lite';

> import { InstantSearch, SearchBox, Hits } from 'react-instantsearch-web';

Reviewer: ```suggestion
import { InstantSearch, SearchBox, Hits } from 'react-instantsearch-hooks-web';
```
